randovania/games/samus_returns/generator/pool_creator.py

```diff
@@ -42,6 +43,22 @@ def pool_creator(results: PoolResults, configuration: BaseConfiguration, game: G
 
 
 def artifact_pool(game: GameDescription, config: MSRArtifactConfig) -> PoolResults:
+    # Check whether we have valid artifact requirements in configuration
+    max_artifacts = 0
+    if config.prefer_metroids:
+        max_artifacts += 25
+    if config.prefer_stronger_metroids:
+        max_artifacts += 14
+    if config.prefer_bosses:
+        if max_artifacts == 39:
+            pass
+        else:
+            max_artifacts += 3
```

Comment:
If you just `pass` when the condition is `true` and just doing things in `else`, just negate the condition in the if-clause.